docs/deploy.md

```diff
@@ -120,25 +118,17 @@ $ api/run.sh &
 
 3. `conf.etcd.endpoints` is used to set ETCD's instances address, it supports multiple instances mode.
 
-```json
-{
-  "etcd": {
-    "endpoints": "127.0.0.1:2379,127.0.0.1:3379"
-  }
-}
+```yaml
+conf:
+  etcd:
+    endpoints:
+      - 127.0.0.1:2379
+      - 127.0.0.1:3379
 ```
 
 ## NOTE
 
-1. When the manager-api is running in background, before we want to rebuild & re-deploy it, we should find the process id then kill it.
-
-```sh
-$ ps aux | grep manager-api
-
-$ kill $process_id
-```
-
-2. After compiling the Manager API, if you move the compiled product to another location, an error will be reported at startup, this is because the configuration file's **absolute path** is fixed in the product and needs to be resolved by running an environment variable to set the location of the configuration file before running.
+1. After compiling the Manager API, if you move the compiled product to another location, an error will be reported at startup, this is because the configuration file's **absolute path** is fixed in the product and needs to be resolved by running an environment variable to set the location of the configuration file before running.
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/98618888-0cd20600-233d-11eb-90fb-fbcbb783651a.png)

Reply:
change to manager-api